core/target-systems/sql-target-system/src/main/java/io/flamingock/targetsystem/mysql/SqlTargetSystemAuditMarker.java

```diff
@@ -37,22 +39,25 @@ public class SqlTargetSystemAuditMarker implements TargetSystemAuditMarker {
     private final String tableName;
     private final DataSource dataSource;
     private final TransactionManager<Connection> txManager;
+    private final SqlAuditMarkerDialectUtils dialectUtils;
 
     public static Builder builder(DataSource dataSource, TransactionManager<Connection> txManager) {
         return new Builder(dataSource, txManager);
     }
 
-    public SqlTargetSystemAuditMarker(DataSource dataSource,
+    private SqlTargetSystemAuditMarker(DataSource dataSource,
                                       String tableName,
-                                      TransactionManager<Connection> txManager) {
+                                      TransactionManager<Connection> txManager,
+                                      SqlAuditMarkerDialectUtils dialectUtils) {
```

Comment:
If we use the class `SqlAuditMarkerDialectUtils`with instances, we should rename it to `SqlAuditMarkerDialectHelper`.